.github/workflows/publish.yml

```diff
@@ -69,12 +69,12 @@ jobs:
       - name: set up .NET
         uses: Brightspace/third-party-actions@actions/setup-dotnet
         with:
-          dotnet-version: 7.0.x
+          dotnet-version: 8.0.100-preview.5.23303.2
```

Comment:
does this not work?
```suggestion
          dotnet-version: 8.0.x
```

Reply:
the project only specifies net8.0 though. if you were thinking of preview package versions - they're independent from the SDK or runtime version